services/director-v2/tests/unit/test_core_settings.py

```diff
@@ -36,6 +36,7 @@ def test_create_registry_settings(project_env_devel_environment, monkeypatch):
     assert settings.api_url == "https://registry:5000/v2"
 
 
+@pytest.mark.skip(reason="registry will not be used from director-v2")
```

Comment:
well maybe just remove this test?

Reply:
well I think you are removing it. I don't want to create more merge conflicts than needed.

Reply:
ok removing it. removal is easier in conflicts

Reply:
ok removing it. removal is easier in conflicts